data/xml/2021.naacl.xml

```diff
@@ -5895,8 +5897,10 @@
       <author><first>James</first><last>Pustejovsky</last></author>
       <pages>76–87</pages>
       <abstract>We propose semantic visualization as a linguistic visual analytic method. It can enable exploration and discovery over large datasets of complex networks by exploiting the semantics of the relations in them. This involves extracting information, applying parameter reduction operations, building hierarchical data representation and designing visualization. We also present the accompanying COVID-SemViz a searchable and interactive visualization system for knowledge exploration of COVID-19 data to demonstrate the application of our proposed method. In the user studies, users found that semantic visualization-powered COVID-SemViz is helpful in terms of finding relevant information and discovering unknown associations.</abstract>
-      <url hash="964dc194">2021.naacl-srw.11</url>
+      <url hash="2e0108f5">2021.naacl-srw.11</url>
       <doi>10.18653/v1/2021.naacl-srw.11</doi>
+      <revision id="1" href="2021.naacl-srw.11v1" hash="964dc194"/>
+      <revision id="2" href="2021.naacl-srw.11v2" hash="2e0108f5" date="2021-06-28">Added a sponsor to Ack section</revision>
```

Comment:
I would probably expand this, e.g., "Added a sponsor to the Acknowledgments section."

Reply:
done